src/assets/js/battle.js

```diff
@@ -0,0 +1,183 @@
+window.addEventListener('load', () => {
+  // create the enemies
+  const goblin = new Enemy('Goblin', 70, 70, 50);
+  const elf = new Enemy('Elf', 100, 100, 70);
+  const giant = new Enemy('Giant', 200, 200, 125);
+  const rat = new Enemy('Rat', 30, 30, 10);
+
+  const playerData = JSON.parse(localStorage.getItem('playerData'));
+  const {
+    name, className, weapon, armor
+  } = playerData;
+
+  let player = null;
+  // Switch statement to assign the player data to a class object
+  switch (className) {
+    case 'mage':
+      if (weapon === 'fire') {
+        player = new Warrior(name, 200, 200, new Weapon('Fire', 60), armor);
+      } else if (weapon === 'ice') {
+        player = new Warrior(name, 200, 200, new Weapon('Ice', 45), armor);
+      } else if (weapon === 'lightning') {
+        player = new Warrior(name, 200, 200, new Weapon('Lightning', 50), armor);
+      }
+      break;
+    case 'knight':
+      if (weapon === 'sword') {
+        player = new Warrior(name, 250, 250, new Weapon('Sword', 45), armor);
+      } else if (weapon === 'dagger') {
+        player = new Warrior(name, 250, 250, new Weapon('Dagger', 25), armor);
+      } else if (weapon === 'war axe') {
+        player = new Warrior(name, 250, 250, new Weapon('War Axe', 30), armor);
+      }
+      break;
+    case 'war beast':
+      if (weapon === 'long sword') {
+        player = new Warrior(name, 250, 250, new Weapon('Long Sword', 45), armor);
+      } else if (weapon === 'battle axe') {
+        player = new Warrior(name, 250, 250, new Weapon('Battle Axe', 40), armor);
+      } else if (weapon === 'claymore') {
+        player = new Warrior(name, 250, 250, new Weapon('Claymore', 60), armor);
+      }
+      break;
+    default:
+      throw new Error('Cannot create player');
+  }
+  // Function to handle a dead player
+  const deadPlayer = (play) => {
+    document.querySelector('#atkMessage').innerHTML = `${play.name} has died`;
+    document.querySelector('#atkBtn').style.display = 'none';
+    document.querySelector('#fleeBtn').style.display = 'none';
+  };
+  // Load the page with the initial player data
+  const initialLoad = (play, gob, el, gi, ra) => {
+    console.log('Player', play);
```

Comment:
I know this is on the front-end, so try to only use logs when you need the information.

Reply:
I'll definitely go through and remove those.  I forgot they were actually there honestly.